docs/reference/commandline/network_create.md

```diff
@@ -14,18 +14,37 @@ parent = "smn_cli"
 
     Creates a new network with a name specified by the user
 
-      -d, --driver=      Driver to manage the Network
+      -d, --driver=[type]      Driver to manage the network
       --help=false       Print usage
 
-Creates a new network that containers can connect to. If the driver supports multi-host networking, the created network will be made available across all the hosts in the cluster. Daemon will do its best to identify network name conflicts. But its the users responsibility to make sure network name is unique across the cluster. You create a network and then configure the container to use it, for example:
+Creates a new network. Driver accepts any of the following `type` values:
+
+**MADHU: Not sure the list of driver types is valid. I have outstanding questions in the Google Doc.**
+
+| Value     | Description                                                                                                                                                                         |
+|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `bridge`  | Uses Linux bridging and iptables to connect containers. This creates a single bridge, called `docker0` by default, and attaches a veth pair between the bridge and every container. |
+| `null`    | A noop implementation of the driver API, used only in cases where no networking is desired. This is to provide backward compatibility to the Docker's --net=none option.            |
```

Comment:
Also wondering if "noop" is something we should use here (and if, is it "no-op"?)